lib/engines/hckinstall/hckinstall.rb

```diff
@@ -241,6 +241,35 @@ def run_second(client:)
       end
     end
 
+    def read_password_from_args
+      args_file = "#{@hck_setup_scripts_path}/args.ps1.example"
```

Comment:
No need to parse `args.ps1.example`. This is just an example and in this case we still duplicate knowlage about default password. Let's leave it only in `config.json`.